packages/react-google-maps-api/src/components/drawing/Marker.tsx

```diff
@@ -7,6 +7,7 @@ import { HasMarkerAnchor } from '../../types'
 
 import { Clusterer } from '@react-google-maps/marker-clusterer'
 import { MarkerClusterer as GoogleClusterer} from '@googlemaps/markerclusterer'
+import * as React from 'react'
```

Comment:
please use destructuring. since react@17 we don't havre to have React  import

Reply:
Done